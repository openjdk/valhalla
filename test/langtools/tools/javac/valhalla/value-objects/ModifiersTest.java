/*
 * @test /nodynamiccopyright/
 * @bug 8279840
 * @summary [lworld] Inconsistent treatment of repeated modifiers.
 * @compile/fail/ref=ModifiersTest.out -XDrawDiagnostics -XDdev ModifiersTest.java
 */

public class ModifiersTest {

    static static class StaticTest {
    }

    native native class NativeTest {
    }

    value value primitive class ValueTest {
    }

    primitive primitive value class PrimitiveTest {
    }

}
