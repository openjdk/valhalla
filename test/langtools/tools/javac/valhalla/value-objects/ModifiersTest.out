ModifiersTest.java:10:12: compiler.err.repeated.modifier
ModifiersTest.java:13:12: compiler.err.repeated.modifier
ModifiersTest.java:16:11: compiler.err.repeated.modifier
ModifiersTest.java:19:15: compiler.err.repeated.modifier
4 errors
